ebean-core/pom.xml

```diff
@@ -239,7 +239,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
         <configuration>
-          <doctitle>Ebean 12</doctitle>
+          <doctitle>Ebean 14</doctitle>
           <overview>src/main/java/io/ebean/overview.html</overview>
```

Comment:
This configuration is not used, when javadoc is called for other modules (`javadoc' has not been previously called for the module: 'io.ebean:ebean-api:jar`)

Reply:
Yeah, this can be removed really. Good catch.